onboarding/mentors.md

```diff
@@ -25,16 +25,14 @@ It's up to the mentoring pair to find what works best for them and to ensure the
 - Keep tabs on your mentee's schedule. Their first day will be full of paperwork and meetings with People Ops but some good tasks for a mentor:
   - Introduce yourself
   - Set a time for an [introductory coffee or tea](#your-first-meeting)
-  - Get in touch with the mentee's manager
-  - (Re-)familiarize yourself with people resources across the company that will be important to your mentee:
-    - Point people in Workplace & People Ops
-    - Their manager
-    - Their team lead
-    - Artsy atlas, calendars, benefit websites, mail filters, Jira and Github management tools...
+  - Introduce the engineering team activities (Monday stand-up, Lunch & Learn, Demo Days, etc.)
+  - Introduce the on-call process
```

Comment:
There's a separate, more in-depth version of this as part of the onboarding checklist. Is the goal of this item to just let the person know that there _is_ an on-call process? (If so we should be specific 😄 )

Reply:
The checklist's section on this subject is perfect. I'll remove this.